source/opt/scalar_replacement_pass.cpp

```diff
@@ -175,7 +175,7 @@ bool ScalarReplacementPass::ReplaceWholeDebugDeclare(
     Instruction* added_dbg_value =
         context()->get_debug_info_mgr()->AddDebugValueForDecl(
             dbg_decl, /*value_id=*/var->result_id(),
-            /*insert_before=*/var->NextNode());
+            /*insert_before=*/var->NextNode(), nullptr);
```

Comment:
The comments for `AddDebugValueForDecl` does not say what happens if `nullptr` is passed at the scope_and_line.